orchagent/dash/dashvnetorch.cpp

```diff
@@ -430,7 +430,7 @@ bool DashVnetOrch::addVnetMap(const string& key, VnetMapBulkContext& ctxt)
             SWSS_LOG_INFO("Not creating VNET map for %s since VNET %s doesn't exist", key.c_str(), ctxt.vnet_name.c_str());
             return false;
         }
-        return addOutboundCaToPa(key, ctxt) && addPaValidation(key, ctxt);
+        return addOutboundCaToPa(key, ctxt) || addPaValidation(key, ctxt);
```

Comment:
I think this should still be "AND" and not "OR".

If let's say addPaValidation returns true and addOutboundCaToPa returns false, we don't want the entry to be removed from the consumer.